backend/controller/controller.go

```diff
@@ -313,7 +313,9 @@ func New(ctx context.Context, conn *sql.DB, config Config, devel bool, runnerSca
 
 func (s *Service) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	start := time.Now()
-
+	ctx, span := observability.Ingress.BeginSpan(r.Context(), r.Method, r.URL.Path)
```

Comment:
For HTTP we should use https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp instead - it should export spans etc. with "standard" naming etc.

Reply:
done